build.md

```diff
@@ -0,0 +1,145 @@
+# Building Volt
+
+###### Note: This repository can be built with the default SDK but will not be outlined in this guide.
+###### Note: x64 platform builds will also not be outlined in this guide.
+
+All build instructions start the same. Clone a local copy of [mini-source-sdk](https://github.com/Joshua-Ashton/mini-source-sdk). (A minified version of [Source SDK](https://github.com/ValveSoftware/source-sdk-2013) with support for VS 2022)
+```bash
+git clone https://github.com/Joshua-Ashton/mini-source-sdk.git
+```
+
+Following your clone of the mini-source-sdk, navigate to the "version" you would like to build (`sdk2013-mp`, `sdk2013-sp`, `asw`). As an example, the multiplayer version will be used from now on.
+```bash
+cd mini-source-sdk/sdk2013-mp/src/
+```
+
+At this time you should now recursively git clone the VPhysics-Jolt repository into your "src" directory.
+```bash
+git clone https://github.com/Joshua-Ashton/VPhysics-Jolt.git vphysics_jolt --recursive
+```
+
+<strong>From this point onward, please follow the rest of the guide specific to your chosen build platform. The current working directory is also assumed to be `mini-source-sdk/sdk2013-mp/src/` and should be adjusted to suit your own build if need be.</strong>
+
+## Windows
+
+### CLI
+
+Continuing on windows, the next step is to the registry fix script (as administrator). This can be done by right-clicking the `fix_registry.bat` file located in `mini-source-sdk/sdk2013-mp/src/`. If your command prompt or powershell instance are already running under administrator just run the following:
```

Comment:
You can just Right Click -> Run as Administrator

Reply:
I swore I wrote that part in, this will get fixed too